frontend/src/hooks/CreateReservation/useCreateReservation.ts

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
+import { useState } from 'react';
+import { useMutation } from 'react-query';
+import { useRecoilValue } from 'recoil';
+import { BASE_URL } from '../../constants';
+import { authAtom } from '../../recoil/atom';
+
+const useCreateReservation = () => {
+  const [date, setDate] = useState('');
+  const { accessToken } = useRecoilValue(authAtom);
+  const isFilled = date;
+
+  const { mutate } = useMutation((Date: string) =>
+    axios({
+      method: 'post',
+      url: `${BASE_URL}/api/reservations`,
+      headers: { Authorization: `Bearer ${accessToken}` },
+      data: {
+        couponId: '1',
+        startAt: Date,
+      },
+    })
+  );
+
+  const setReservationDate = e => {
+    setDate(e.target.value + ` 00:00:00`);
+  };
+
+  const sendReservation = () => {
+    mutate(date);
+  };
+
+  return { isFilled, setReservationDate, sendReservation };
```

Comment:
```suggestion
  return { isFilled:date, setReservationDate, sendReservation };
```
위에 변수를 지우고 이렇게 가도 되지 않을까요?  `isFilled.length` `!!isFilled`도 괜찮을 것 같고...